devdocker

```diff
@@ -183,7 +183,7 @@ RUN mkdir -p $SRCDIR
     ]
     for var, value in cfg.environment.items():
       envFlags.extend(['-e', '%s=%s' % (var, value % substitutions)])
-    mountFlags = [ '-v', '%s:%s' % (srcdir, srcdir) ]
+    mountFlags = [ '-v', '%s:%s:delegated' % (srcdir, srcdir) ]
```

Comment:
it may be helpful to provide this as a flag in `devdocker create` so we can experiment with this setting for a while before making this the default.